ZIGSIMPlus/Models/Services/ServiceManager.swift

```diff
@@ -126,4 +126,12 @@ class ServiceManager {
             "sensordata": data
         ])
     }
+
+    public func getErrorLog() -> String {
```

Comment:
For me, it's more natural to return `String?` and nil when no error, because many Swift APIs adopt such form (not in errorLog but in error, though).
https://developer.apple.com/documentation/avfoundation/avplayeritem/1387573-errorlog
What do you think?

Reply:
Sounds reasonable. I chose `String` because I don't wanted to let presenters concern about if the error is nil or not, but it had to check `errorLog ==""`...
Anyways, I fixed it to return `String?` in 2f30b81. Please check it out 🙏